linux_new/jdk/alpine/src/main/packaging/Dockerfile

```
@@ -1,6 +1,6 @@
 ARG CONTAINER_REGISTRY=""
 
-FROM ${CONTAINER_REGISTRY}alpine:3.19
+FROM ${CONTAINER_REGISTRY}alpine
```

please keep this pinned, just bump to the latest version

```suggestion
FROM ${CONTAINER_REGISTRY}alpine:3.21
```